app/views/responsible_body/mobile/extra_data_requests/index.html.erb

```diff
@@ -5,28 +5,32 @@
       <%= link_to t('page_titles.responsible_body_home'), responsible_body_home_path, class: 'govuk-breadcrumbs__link' %>
     </li>
     <li class="govuk-breadcrumbs__list-item">
-      <%= t('page_titles.request_extra_mobile_data') %>
+      <%= t('page_titles.request_extra_data_for_mobile_devices') %>
     </li>
   </ol>
   </div>
 <% end %>
-
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      <%= t('page_titles.request_extra_mobile_data') %>
+      <%= t('page_titles.request_extra_data_for_mobile_devices') %>
     </h1>
-
-    <% unless @extra_mobile_data_requests.any? %>
-      <%= render partial: 'shared/use_the_mobile_guide' %>
-    <% end %>
```

Comment:
This has changed the design back to an earlier one.

Instead it should:

- only show guidance when there are no requests
- show the `use_the_mobile_guide` partial
- the updated button text is correct though

It should look like this:

![Screen Shot 2020-08-03 at 14 47 54](https://user-images.githubusercontent.com/319055/89189763-5bcb0b00-d598-11ea-811b-4a5a9cfa1751.png)

Reply:
OK done. I seem to have not got to grips with where I should be looking for the designs that I should be implementing. I'd assumed the prototype was the place to look, but it doesn't look like this?

Reply:
@tonyheadford Sorry, this is my bad, I'd made a design update directly in this repo, without first updating the prototype.